src/server/components/lists/searches/lawyers.ts

```diff
@@ -25,23 +24,56 @@ export async function searchLawyers(
   res: Response
 ): Promise<void> {
   const params = getAllRequestParams(req);
-  const { serviceType, country, legalAid, region, proBono } = params;
-  const practiceArea = parseListValues("practiceArea", params);
+  const { serviceType, country, region, page } = params;
+  let practiceArea = parseListValues("practiceArea", params);
+  if (practiceArea != null) {
+    practiceArea = practiceArea.map(area => area.toLowerCase());
+  }
+  let pageNum;
+  if (page === undefined || page === null || page === "") {
+    pageNum = 1;
+
+  } else {
+    pageNum = parseInt(page);
+  }
```

Comment:
```
let pageNum = page;
pageNum ||= 1  // if pageNum is falsy e.g. undefined, null, empty string, assign 1
```
should be ok if page has come through as "0" or "1" since stringified 0 or 1 are both truthy

Reply:
the issue here is we need to check page.  I've refactored this section and applied the same technique to the page variable.  Cheers.